### ISSUE TITLE
Result of cloudflare endpoint fetch has no "data" atttibute in JSON

### ISSUE THREAD

**Issue by philschonholzer** [zentered/cloudflare-preview-url#60]
**Describe the bug**
I always got:
```sh
Fetching from: https://api.cloudflare.com/client/v4/accounts/***/pages/projects/groupify/deployments
Error: no deployments found
Error: no deployments found
```

**To Reproduce**
I copied part of the code locally and run it:

```ts
async function getDeployment() {
       // Add accountId
	const apiUrl = `https://api.cloudflare.com/client/v4/accounts/***/pages/projects/groupify/deployments`

	const headers = {
                 // Add token
		Authorization: `Bearer xxx`,
	}

	const res = await fetch(apiUrl, { headers })
       // with {data) it would always throw, so I don't access the "data" attribute and it works
	const data = await res.json()

	if (!data || !data.result || data.result.length <= 0) {
		throw new Error('no deployments found')
	}
	console.log('data', data)
}

getDeployment()

```

**Expected behavior**
Should be able to get the deployments

**Comment by philschonholzer** [comment]
I now using this action and it worked: https://github.com/WalshyDev/cf-pages-await

**Comment by PatrickHeneise** [comment]
Thanks @philschonholzer for letting me know. I didn't realized I messed up the `.json()` data there 💩

Fixed the bug in #61
